Cargo.toml

```diff
@@ -29,3 +29,4 @@ async-std = { version = "1.3", optional = true, features = ["unstable"] }
 tokio = { version = "0.2", optional = true, features = ["rt-core", "blocking", "rt-util"] }
 # core = { package = "core-futures-tls", version = "0.1.0", optional = true }
 lightproc = { version = "0.3.4", optional = true }
+atomic_refcell = "0.1.6"
```

Comment:
if you are using this dependency in only feature-gated places, then it can be marked optional and that is depending on that feature

Reply:
Right, totally forget that